src/routes/choice-exercises/components/SingleChoiceExercise.tsx

```diff
@@ -95,6 +96,10 @@ const ChoiceExerciseScreen = ({
   }
   const count = documents.length
 
+  const onExerciseCheated = async (result: SimpleResult): Promise<void> => {
+    await onExerciseFinished(results.map(it => ({ ...it, numberOfTries: 1, result })))
```

Comment:
(optional)  I think it would be better to set numberOfTries to `numberOfMaxRetries`, as it is not a result that can happen in the real app, that there is only 1 try, but wrong. 
(oke currently it is, when clicking on "show solution", but this will be removed.)

Reply:
changed